db/db.go

```diff
@@ -94,7 +82,8 @@ func (w *WorkspaceDB) InitTables() error {
 				member_group TEXT NOT NULL,
 				role_name TEXT NULL,
 				role_arn TEXT NULL,
-				status TEXT NOT NULL
+				status TEXT NOT NULL,
+				last_updated TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL
```

Comment:
For last_updated, can this cause any race condition errors? E.g. update A (earlier) and update B (later) sent from source, update A gets chosen by a consumer A but takes longer to process, update B is actioned first. Update A would therefore not be processed. Is that definitely the desired behaviour? 

I guess if updates always have the complete state of the workspace then it is fine, just confirming.

Reply:
That is the desired behavior yes. Whatever is consumed will always be the true reflection of the workspace status at that point in time.